src/ui/popup.js

```diff
@@ -231,6 +234,22 @@ export default class Popup extends Evented {
         return this.setDOMContent(frag);
     }
 
+    /**
```

Comment:
This has to be moved above the `setMaxWidth` (or `set maxWidth` if you go the getter/setter route) method. You should also add a doc comment for `getMaxWidth`.